### PR TITLE
Brush up deity Minor Boons

### PR DESCRIPTION
**Pull request by simonward**
Let me know if you prefer different names for the effects "Abadar's Warning", "Rovagug's Destruction", "Shizuru's Light", and "Torag's Repairs".

### CODE REVIEW

File: packs/boons-and-curses/abadar-minor-boon.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "deityboon",
         "description": {
-            "value": "<p><strong>Minor Boon (Single Use)</strong> Abadar warns his favored against those who might unfairly take advantage. Once, when someone rolls a success on a Deception check to Lie maliciously to you and you alone, they get a critical failure instead.</p>\n<p>Abadar typically chooses to grant this boon in response to an extremely consequential lie.</p>"
+            "value": "<p>Abadar warns his favored against those who might unfairly take advantage.</p>\n<p>Once, when someone rolls a success on a Deception check to @UUID[Compendium.pf2e.actionspf2e.Item.Lie] maliciously to you and you alone, they get a critical failure instead.</p>\n<p>Abadar typically chooses to grant this boon in response to an extremely consequential lie.</p>"
```
Comment (SpartanCPA) [inline packs/boons-and-curses/abadar-minor-boon.json:14]: Can you make a roll note for this? Lie is a secret action, so the Note should only show up for the GM, but they should be notified that the book could activate.

File: packs/boons-and-curses/achaekek-minor-boon.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "name": "Achaekek - Minor Boon",
     "system": {
         "description": {
-            "value": "<p>Once, when you would fail a Stealth check, you critically succeed instead. Achaekek typically grants this boon for an extremely consequential Stealth check that could lead to an assassination, such as one that could get you into position to kill an important target, but rarely on a Stealth check to help you escape.</p>"
+            "value": "<p>Once, when you would fail a Stealth check, you critically succeed instead.</p>\n<p>Achaekek typically grants this boon for an extremely consequential Stealth check that could lead to an assassination, such as one that could get you into position to kill an important target, but rarely on a Stealth check to help you escape.</p>"
```
Comment (SpartanCPA) [inline packs/boons-and-curses/achaekek-minor-boon.json:7]: Can you make a roll note for this? Sneak/Hide are secret actions, so the Note should only show up for the GM, but they should be notified that the boon could activate.

File: packs/boons-and-curses/achaekek-minor-boon.json [hunk 3]
```diff
@@ -20,7 +20,18 @@
             "remaster": true,
             "title": "Pathfinder Adventure: Prey for Death"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "outcome": [
+                    "failure"
```
Comment (simonward) [inline packs/boons-and-curses/achaekek-minor-boon.json:27]: It specifies failure, but doesn't mention critical failure.

File: packs/boons-and-curses/abadar-minor-boon.json (hunk 1 above)
Comment (simonward) [reply packs/boons-and-curses/abadar-minor-boon.json:14]: I'm sure we can, since you're the target of the lie rather than the liar.

Comment (TikaelSol) [reply packs/boons-and-curses/abadar-minor-boon.json:14]: We can ephemeral effect the note

**Pull request merged**